app/src/main/res/layout/mark_stories_completed_fragment.xml

```diff
@@ -114,6 +115,7 @@
         android:layout_height="match_parent"
         android:background="@color/profileEditBackground"
         android:clipToPadding="false"
+        android:contentDescription="@string/mark_stories_completed_recycler_view"
```

Comment:
Missed this earlier. We should not add `contentDescription` to `RecyclerView`

Reply:
![image](https://user-images.githubusercontent.com/63469151/127130439-8807e7de-44ac-4fd5-9ff0-4218c406afb8.png)

I was getting this error on running the tests on Espresso and adding `contentDescription` to the recycler view fixed it. Is it some kind of bug? Or did I misunderstood the error?

Reply:
Same for the one also (#3469).

Reply:
This error is because of when this test is running there is no content inside recyclerview.

Instead of doing this solution, try to update the test case such that there is content inside recyclerview and this error won't appear in that case.

@yashraj-01

Reply:
This error is because of when this test is running there is no content inside recyclerview.

Instead of doing this solution, try to update the test case such that there is content inside recyclerview and this error won't appear in that case.

@yashraj-01

Reply:
![Screenshot from 2021-07-29 18-46-38](https://user-images.githubusercontent.com/63469151/127498995-d2516b5c-3854-46c7-95bf-22eda1aa48e4.png)
@rt4914 it seems like this was an issue with the emulator as when I ran the Espresso tests on a physical device, all the tests passed successfully.

Reply:
So, I have removed the `contentDescription` from the recyclerview.

Reply:
All tests are passing.